lib/views/go_to_page.dart

```diff
@@ -0,0 +1,174 @@
+                      suggestionsCallback: (pattern) async {
+                        var roomTitles = new List<String>();
+                        //TODO: add room list here
+                        for (var room in []) {
+                          if (room
+                              .getTitle()
+                              .toUpperCase()
+                              .startsWith(pattern.toString().toUpperCase())) {
+                            roomTitles.add(room.getTitle());
+                          }
+                        }
+                        return roomTitles; //await BackendService.getSuggestions(pattern);
+                      },
+                      itemBuilder: (context, suggestion) {
+                        return Card(
+                          elevation: 2,
+                          child: ListTile(
+                            leading: Icon(Icons.place,
+                                color: Color.fromRGBO(147, 0, 47, 1)),
+                            title: Text(suggestion),
+
+                            //subtitle: Text('\$${suggestion['price']}'),
+                          ),
+                        );
+                      },
+                      onSuggestionSelected: (suggestion) {
+                        this._originTypeAheadController.text = suggestion;
+                        originDestination.add("$suggestion");
+                        FocusScope.of(context).requestFocus(
+                            destinationTextField); // to change the focus of the textfield to the second textfield
+                      },
+                    ),
+                  ),
+                ],
+              ),
+              SizedBox(height: 10.0),
+              Row(
+                children: <Widget>[
+                  Expanded(flex: 1, child: Text('')),
+                  Expanded(
+                    flex: 10,
+                    child: TypeAheadField(
+                      textFieldConfiguration: TextFieldConfiguration(
+                        focusNode:
+                            destinationTextField, //set the focus to the variable defined in definition of class
+                        style: TextStyle(
+                          fontSize: 18,
+                          fontWeight: FontWeight.w400,
+                          color: Colors.black,
+                        ),
+                        decoration: InputDecoration(
+                          contentPadding:
+                              EdgeInsets.fromLTRB(15.0, 12.0, 10.0, 12.0),
+                          hintText: "destination...",
+                          border: OutlineInputBorder(
+                              borderRadius: BorderRadius.circular(10.0)),
+                        ),
+                        controller: this._destinTypeAheadController,
+                      ),
+                      suggestionsCallback: (pattern) async {
+                        var roomTitles = new List<String>();
```

Comment:
Most likely going to want to make this a List of RoomCoordinate. You should also add a TODO comment similar to line 78